mobile/src/main/java/com/gelakinetic/mtgfam/helpers/database/CardDbAdapter.java

```diff
@@ -548,7 +556,8 @@ public static Cursor fetchCards(long[] ids, String orderByStr, SQLiteDatabase da
      * @throws FamiliarDbException If something goes wrong
      */
     public static Cursor fetchCardByName(String name, List<String> fields, boolean shouldGroup,
-                                         boolean offlineOnly, boolean preferOptionalFoil, SQLiteDatabase mDb)
+                                         boolean offlineOnly, boolean preferOptionalFoil,
+                                         SQLiteDatabase mDb, Set<String> languages)

```

Comment:
`languages` should also be added to javadoc

Reply:
You're right. Thanks.